components/eam/bld/build-namelist

```diff
@@ -1119,6 +1119,10 @@ if ($aer_model eq 'mam' ) {
   if ($chem =~ /_mam4/) {$aero_modes = '4mode';}
   if ($chem =~ /_mam4_mom/) {$aero_modes = '4mode_mom';}
   if ($chem =~ /_mam4_resus_mom/) {$aero_modes = '4mode_mom';}
+# kzm ++
+  if ($chem =~ /_mam7_resus_mom_s/) {$aero_modes = '7mode_s';}
+  if ($chem =~ /_mam5_resus_mom/) {$aero_modes = '5mode';}
+# --  
```

Comment:
You may want to delete those comments with "kzm ++" and "kzm--". I think there a lot of places have those comments.